src/main/java/org/ohdsi/circe/cohortdefinition/CohortExpressionQueryBuilder.java

```diff
@@ -634,8 +640,13 @@ public String getCorelatedlCriteriaQuery(CorelatedCriteria corelatedCriteria, St
       getOccurrenceOperator(corelatedCriteria.occurrence.type), 
       corelatedCriteria.occurrence.count
     );
+		
+		// join type is LEFT when counts of 0 or 'at most' is specified
+		String joinType = (corelatedCriteria.occurrence.type == 1 || corelatedCriteria.occurrence.count == 0) ?  "LEFT" : "INNER";
```

Comment:
Where is the type value (1) coming from? - would you mind to move it into constants (or re-use existing one)?